sim/option.go

```diff
@@ -69,6 +71,12 @@ func NewOptions(opts []Option) *Options {
 		o.OutputDir = filepath.Join(homeDir, ".config", "simnet")
 	}
 
+	err := os.Mkdir(o.OutputDir, 0755)
```

Comment:
We can keep it like that for now. But we should eventually update that function to return an error.

Reply:
Agreed